include/wavm/WAVMWasmModule.h

```diff
@@ -138,21 +139,15 @@ class WAVMWasmModule final
     WAVM::Runtime::GCPointer<WAVM::Runtime::Instance> wasiModule;
     WAVM::Runtime::GCPointer<WAVM::Runtime::Instance> moduleInstance;
 
-    // Dynamic modules
-    int dynamicModuleCount = 0;
-    int nextMemoryBase = 0;
-    int nextStackPointer = 0;
-    int nextTableBase = 0;
```

Comment:
Tracking the values of these `int` properties was difficult, so I've refactored the logic into the `LoadedDynamicModule` class